crates/telio-lana/src/event_log_moose.rs

```diff
@@ -59,34 +59,35 @@ pub fn init_context_info() {
     let foreign_tracker = "nordvpnapp";
 
     if let Ok(foreign_context) = moose::fetch_specific_context(foreign_tracker) {
-        let (device_info, user_info) = parse_foreign_context(&foreign_context);
+        let tracker_context_info = parse_foreign_context(&foreign_context);
```

Comment:
nit: if `device_info` wouldn't be renamed, the resulting diff could possibly be easier to follow in two commits: one to add the new field, another one to rename